src/cascadia/TerminalConnection/packages.config

```diff
@@ -1,4 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Windows.CppWinRT" version="2.0.190417.3" targetFramework="native" />
-</packages>
+  <package id="vcpkg-export-20190703-153703" version="1.0.0" targetFramework="native" />
+  <package id="vcpkg-export-20190703-153928" version="1.0.0" targetFramework="native" />

```

Comment:
It seems wrong to have two versions of this in here.

Reply:
Github was unhappy with accepting a file greater than 100 MB, so I split the package into 2 (one for x86 and one for x64), thats why there's 2. This will be solved when we figure out a way to make the package smaller (probably with static linking according to Dustin).

Reply:
Resolved in latest pull request.